translate_templates.py

```diff
@@ -2,6 +2,7 @@
 
 import os
 import os.path as osp
```

Comment:
If you start using pathlib, you can take this opportunity to get rid of os.path ? Up to you.

Reply:
Yes agreed. Even if you don t do it in the whole library, I think it is great to keep only one of them at the module level. 
I just checked and os.rename is compatible with Path. I don t know about `dir_util.copy_tree` tho.